### PR TITLE
HACDOCS-470: Editing Quarkus code sample readme.

### PR DESCRIPTION
**Pull request by jseseCCS**
[HACDOCS-470](https://issues.redhat.com/browse/HACDOCS-470) editing README content for Quarkus code sample.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,18 @@
...
-## Running the application in dev mode
+* [Quarkus `devfile.yaml`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml)
+* [Quarkus `Dockerfile`](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/docker/Dockerfile)
 
-You can run your application in dev mode that enables live coding using:
-```shell script
-./mvnw compile quarkus:dev
-```
+1. The `devfile.yaml` file has an [`image-build` component](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/devfile.yaml#L20-L27) that points to your `Dockerfile`.
+2. The [`docker/Dockerfile`](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/docker/Dockerfile) contains the instructions you need to build the code sample as a container image.
```
Comment (maysunfaisal) [inline README.md:11]:
```suggestion
2. The [`docker/Dockerfile`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/src/main/docker/Dockerfile.jvm.staged) contains the instructions you need to build the code sample as a container image.
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,18 @@
...
+3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/devfile.yaml#L27-L40) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```
Comment (maysunfaisal) [inline README.md:12]:
```suggestion
3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml#L29-L42) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```

File: README.md [hunk 3]
```diff
@@ -1,58 +1,18 @@
...
+4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/devfile.yaml#L42-L55) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```
Comment (maysunfaisal) [inline README.md:13]:
```suggestion
4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml#L44-L57) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```

File: README.md [hunk 4]
```diff
@@ -1,58 +1,18 @@
...
-
-## Provided examples
-
-### RESTEasy JAX-RS example
-
-REST is easy peasy with this Hello World RESTEasy resource.
-
-[Related guide section...](https://quarkus.io/guides/getting-started#the-jax-rs-resources)
+### Additional resources
+* For more information about Go, see [go.dev](https://go.dev/).
```
Comment (maysunfaisal) [inline README.md:16]:
```suggestion
* For more information about Quarkus, see [quarkus.io](https://quarkus.io/).
```

File: README.md [hunk 5]
```diff
@@ -1,58 +1,18 @@
...
 
-This project uses Quarkus, the Supersonic Subatomic Java Framework.
+**Note:** The Quarkus code sample uses the **8081** HTTP port.
 
-If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+Before you begin creating an application with this `devfile` code sample, it's helpful to understand the relationship between the `devfile` and `Dockerfile` and how they contribute to your build. You can find these files at the following URLs:
 
-## Running the application in dev mode
+* [Quarkus `devfile.yaml`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml)
+* [Quarkus `Dockerfile`](https://github.com/devfile-samples/devfile-sample-go-basic/blob/main/docker/Dockerfile)
```
Comment (maysunfaisal) [inline README.md:8]:
```suggestion
* [Quarkus `Dockerfile`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/src/main/docker/Dockerfile.jvm.staged)
```

**Comment by maysunfaisal** [comment]
@jseseCCS I've reviewed this PR for fixing the links

**Comment by jseseCCS** [comment]
hi, @maysunfaisal! thanks for your review! sorry for all the wrong links. i hadn't planned for you to review this one yet. i had copied/pasted the Go readme as a template and was going to come back and replace it with the correct Quarkus info, but i think you got here before I did that! 

can you review again now that i've made all the changes?

**Comment by jseseCCS** [comment]
@maysunfaisal also, can you confirm that the link to the `deploy` command is correct?

File: README.md [hunk 6]
```diff
@@ -1,58 +1,18 @@
...
+* [Quarkus `devfile.yaml`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml)
+* [Quarkus `Dockerfile`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/src/main/docker/Dockerfile.jvm.staged)
 
-You can run your application in dev mode that enables live coding using:
-```shell script
-./mvnw compile quarkus:dev
-```
+1. The `devfile.yaml` file has an [`image-build` component](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml#L22-L28) that points to your `Dockerfile`.
+2. The [`src/main/docker/Dockerfile.jvm.staged`](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/src/main/docker/Dockerfile.jvm.staged) file contains the instructions you need to build the code sample as a container image.
+3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml#L29-L42) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```
Comment (maysunfaisal) [inline README.md:12]:
```suggestion
3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-code-with-quarkus/blob/main/devfile.yaml#L29-L40) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**